frontend-dsl-fabric/src/main/groovy/FabricModsDotGroovy.groovy

```diff
@@ -0,0 +1,327 @@
+import org.jetbrains.annotations.Nullable
+
+/**
+ * This is the Fabric frontend layer
+ */
+@PackageScope
+@CompileStatic
+@Log4j2(category = 'MDG - Fabric Frontend')
+class FabricModsDotGroovy extends ModsDotGroovyFrontend implements PropertyInterceptor, MapClosureInterceptor {
+    /**@
+     * If running in a Gradle environment, this will be populated with the {@code build.properties}.
+     */
+    public final Map<String, ?> buildProperties = [:]
+
+    /**@
+     * Needed for internal mechanisms. Must always be 1.
+     * Mandatory
+     */
+    int schemaVersion = 1
+
+    /**@
+     * Defines the mod's identifier - a string of Latin letters, digits, underscores with length from 2 to 64.
+     * Mandatory
+     */
+    String id = "examplemod"
+
+    /**@
+     * Defines the mod's version - a string value, optionally matching the <a href="https://semver.org/">Semantic Versioning 2.0.0</a> specification.
+     * Mandatory
+     */
+    String version = "1.0.0"
+
+    /**@
+     * Defines the user-friendly mod's name. If not present, assume it matches id.
+     */
+    @Nullable String name = null
+
+    /**@
+     * Defines the mod's description. If not present, assume empty string.
+     */
+    @Nullable String description = null
+
+    /**@
+     * Defines the list of ids of mod. It can be seen as the aliases of the mod. Fabric Loader will treat these ids as mods that exist.
+     * If there are other mods using that id, they will not be loaded.
+     */
+    @Nullable List<String> provides = null
+
+    /**@
+     * Defines where mod runs: only on the client side (client mod), only on the server side (plugin) or on both sides (regular mod). Contains the environment identifier:
+     */
+    @Nullable Environment environment = null
+
+    /**@
+     * Defines the mod's icon. Icons are square PNG files.
+     * (Minecraft resource packs use 128×128, but that is not a hard requirement - a power of two is, however, recommended.)
+     * Can be provided in one of two forms:
+     * <ul>
+     *     <li>A path to a single PNG file.</li>
+     *     <li>A dictionary of images widths to their files' paths.</li>
+     * </ul>
+     */
+    @Nullable String icon = null
+
+    /**@
+     * Defines the licensing information.
+     * This should provide the complete set of preferred licenses conveying the entire mod package. In other words,
+     * compliance with all listed licenses should be sufficient for usage, redistribution, etc. of the mod package as a whole.
+     * For cases where a part of code is dual-licensed, choose the preferred license. The list is not exhaustive,
+     * serves primarily as a kind of hint, and does not prevent you from granting additional rights/licenses on a case-by-case basis.
+     * To aid automated tools, it is recommended to use <a href="https://spdx.org/licenses/">SPDX License Identifiers</a> for open-source licenses.
+     */
+    @Nullable def license = null
```

Comment:
This all gets wrapped into one long paragraph. Might wanna put <p></p> for adding new lines.
<img width="557" alt="image" src="https://github.com/PaintNinja/ModsDotGroovy/assets/3158390/5f671827-40dc-408d-9da0-4103d10684cd">

```suggestion
    /**@
     * Defines the licensing information.
     * <p>This should provide the complete set of preferred licenses conveying the entire mod package. In other words,
     * compliance with all listed licenses should be sufficient for usage, redistribution, etc. of the mod package as a whole.</p>
     * <p>For cases where a part of code is dual-licensed, choose the preferred license. The list is not exhaustive,
     * serves primarily as a kind of hint, and does not prevent you from granting additional rights/licenses on a case-by-case basis.</p>
     * <p>To aid automated tools, it is recommended to use <a href="https://spdx.org/licenses/">SPDX License Identifiers</a> for open-source licenses.</p>
     */
    @Nullable def license = null
```